### PR TITLE
Add refresh button to user pages

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: store/modules/mission.js [hunk 1]
```diff
@@ -125,10 +131,24 @@ const mutations = {
...
+    state.isFetched = false;
+    state.isFetchingMissionHistory = false;
+    state.isFetchedMissionHistory = false;
+  },
+  [MISSION_START_FETCHING_MISSION_LIST](state) {
+    state.isFetching = true;
+  },
+  [MISSION_END_FETCHING_MISSION_LIST](state) {
+    state.isFetching = false;
+    if (!state.isFetched) state.isFetched = true;
```
Comment (williamchong007) [inline store/modules/mission.js]: should omit this `if`

File: store/modules/mission.js [hunk 2]
```diff
@@ -125,10 +131,24 @@ const mutations = {
...
   },
   [MISSION_START_FETCHING_MISSION_HISTORY_LIST](state) {
-    state.isHistoryMissionsFetched = false;
+    state.isFetchingMissionHistory = true;
+  },
+  [MISSION_END_FETCHING_MISSION_HISTORY_LIST](state) {
+    state.isFetchingMissionHistory = false;
+    if (!state.isFetchedMissionHistory) state.isFetchedMissionHistory = true;
```
Comment (williamchong007) [inline store/modules/mission.js]: should omit this `if`

Comment (nwingt) [reply store/modules/mission.js]: Updated

File: assets/index.scss [hunk 3]
```diff
@@ -844,3 +861,14 @@ span.bold {
   @extend .lc-color-like-green;
   @extend .lc-font-weight-600;
 }
+
+
+.simple-svg-wrapper {
+  display: flex;
```
Comment (williamchong007) [inline assets/index.scss]: seems this broke MissionDIalog's icon

Comment (williamchong007) [reply assets/index.scss]: nvm missed https://github.com/likecoin/likecoin-store/pull/427/commits/0397b5f72c786553b96a0dc610e30f13187b1560

**Pull request merged**